setup/setup.iss

```diff
@@ -30,11 +30,10 @@ WizardStyle=modern
 [Tasks]
 Name: "desktopicon"; Description: "{cm:CreateDesktopIcon}"; GroupDescription: "{cm:AdditionalIcons}"
 Name: "rfproto"; Description: "Register rf:// protocol handler"; GroupDescription: "Other options:"
-Name: "rflassoc"; Description: "Associate .rfl file extension with Dash Faction Level Editor"; GroupDescription: "Other options:"
-Name: "fftracker"; Description: "Set the multiplayer tracker to rfgt.factionfiles.com"; GroupDescription: "Other options:"
-Name: "patchgame"; Description: "Install required game patches"; GroupDescription: "Other options:"; Check: "PatchGameTaskCheck"
-Name: "replacerflauncher"; Description: "Replace the Red Faction launcher with the Dash Faction launcher (allows Dash Faction to be opened through Red Faction on Steam)"; GroupDescription: "Other options:"; Flags: unchecked
-Name: "redvisualstyles"; Description: "Enable Windows Visual Styles for the level editor (experimental)"; GroupDescription: "Other options:"; Flags: unchecked
+Name: "rflassoc"; Description: "Associate the .rfl file extension with the level editor"; GroupDescription: "Other options:"
+Name: "patchgame"; Description: "Install necessary game patches"; GroupDescription: "Other options:"; Check: "PatchGameTaskCheck"
+Name: "replacerflauncher"; Description: "Replace the game launcher with the Dash Faction launcher (enables Steam support)"; GroupDescription: "Other options:"; Flags: unchecked

```

Comment:
why changing "Red Faction" to "game" everywhere? I through "Red Faction" would make it more clear like in this case where we have RF launcher and DF launcher

Reply:
I replaced "Red Faction" with "game" in most places because I believed it would already be known to the user that they were applying the patch to RF. Looking at it again, it is possible that they may mistake Dash itself for the "game", as community patches for other games tend to completely replace the original files. I will make adjustments to clarify this.